src/JuliusSweetland.OptiKey.UnitTests/Extensions/DoubleExtensionsTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using JuliusSweetland.OptiKey.Extensions;
+using NUnit.Framework;
+
+namespace JuliusSweetland.OptiKey.UnitTests.Extensions
+{
+    [TestFixture]
+    public class DoubleExtensionsTests
+    {
+        [Test]
+        public void TestCoerceToUpperLimit()
```

Comment:
It is minor but DoubleExtensions.CoerceToUpperLimit and DoubleExtensions.CoerceToLowerLimit could use Math.Min and Math.Max for readability.